src/Microsoft.CodeQuality.Analyzers.Exp/Core/FlowAnalysis/Framework/DataFlow/AbstractLocationDataFlowOperationVisitor.cs

```diff
@@ -57,54 +62,89 @@ protected void ResetCurrentAnalysisData(IDictionary<AbstractLocation, TAbstractA
             }
         }
 
-        protected virtual TAbstractAnalysisValue HandleInstanceCreation(IOperation operation, PointsToAbstractValue instanceLocation, TAbstractAnalysisValue defaultValue)
+        protected virtual TAbstractAnalysisValue HandleInstanceCreation(ITypeSymbol instanceType, PointsToAbstractValue instanceLocation, TAbstractAnalysisValue defaultValue)
         {
             SetAbstractValue(instanceLocation, defaultValue);
             return defaultValue;
         }
 
+        protected override TAbstractAnalysisValue ComputeAnalysisValueForOutArgument(IArgumentOperation operation, TAbstractAnalysisValue defaultValue)
+        {
+            if (operation.Value.Type != null)
+            {
+                PointsToAbstractValue instanceLocation = GetPointsToAbstractValue(operation);
+                return HandleInstanceCreation(operation.Value.Type, instanceLocation, defaultValue);
+            }
+
+            return defaultValue;
+        }
+
+        protected override void SetValueForParameterOnEntry(IParameterSymbol parameter, AnalysisEntity analysisEntity)
+        {
+        }
```

Comment:
likely, there should be a comment confirming that the void override is OK

Reply:
Yes, sure.